app/config/config.go

```diff
@@ -0,0 +1,42 @@
+// Package config works with config file for building pages
```

Comment:
A good comment would answer "why" question alongside with "what": now it's more or less meaningless because without looking into the code you have no idea what is that package purpose.